packages/web/src/features/users/UpdateProfileView.tsx

```diff
@@ -63,11 +70,23 @@ export default function UpdateProfileView() {
 
   if (!user) return null;
 
+  const test = formContext.watch('test');
+
   return (
     <View fitToScreen className="flex justify-center items-start">
       <Card className="mx-4 mt-4 w-96 flex-shrink p-6">
         <ViewTitle>{t('users:update_profile')}</ViewTitle>
         <Form context={formContext} onSubmit={onSubmit}>
+          <div className="mb-2">
+            <FormLabel id="test">TEST</FormLabel>
+            <RichTextInput
+              aria-labelledby="test"
+              {...formContext.register('test', {
+                value: '<p>Hello Worldוָבֹ֔הוּanother</p>',
+              })}
+            />
```

Comment:
After seeing the consequences of embedding the react hook form registration in custom components in the SortableMultiselectInput, I think we should separate that out like this instead.

The RichTextInput component is proof of concept for how we can design our components to work directly with `formContext.register` without needing to use the `Controller` component like we are in the mutliselect components. This way, if you are using react hook form, you can just use register like I am here. If you aren't using react hook form, `RichTextInput` can works like a normal component.

I'll pull this into a separate ticket to work on, but I wanted to see how that could work for a complex component like this one

Reply:
> I'll pull this into a separate ticket to work on

Does that mean you'll modify this PR? Or that you'll rework our other input components separately?